p2p/simulations/http.go

```diff
@@ -260,6 +272,8 @@ func NewServer(config *ServerConfig) *Server {
 	s.GET("/networks", s.GetNetworks)
 	s.GET("/networks/:netid", s.GetNetwork)
 	s.GET("/networks/:netid/events", s.StreamNetworkEvents)
+	s.POST("/networks/:netid/snapshots/create", s.CreateSnapshot)
```

Comment:
> creating a snapshot should be a GET /networks/:netid/snaphot i mean why POST?

A `GET` request should never be "creating" things, we are creating a new snapshot here not getting an existing one, hence `POST`.

> loading should be PUT /networks/:netid/snapshot why POST and 'create' is never part of the url no?

`PUT` is supposed to be "store this resource at this URL", we aren't doing that here, we are mutating the state of the network by essentially providing a diff, so I think `POST` is more appropriate, and also makes it easier to perform from a web browser (e.g. just "upload" a snapshot).